From f5a55412318e7131d084f62e88ca9324d956295f Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Wed, 18 May 2022 03:06:04 -0700
Subject: [PATCH] update  link of zlib-1.2.11.tar.gz

The original one is obsolete, use a new one to replace

Upstream-Status: Pending

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 WORKSPACE | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/WORKSPACE b/WORKSPACE
index 39f478f8..c7ecccad 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -46,7 +46,7 @@ http_archive(
     strip_prefix = "zlib-1.2.11",
     urls = [
         "https://mirror.bazel.build/zlib.net/zlib-1.2.11.tar.gz",
-        "https://zlib.net/zlib-1.2.11.tar.gz",
+        "https://sourceforge.net/projects/libpng/files/zlib/1.2.11/zlib-1.2.11.tar.gz/download",
     ],
 )
 
-- 
2.36.0

